packages/dds/tree/src/test/simple-tree/toMapTree.spec.ts

```diff
@@ -361,11 +469,17 @@ describe("toMapTree", () => {
 								],
 								[
 									brand("bar"),
-									[{ type: leaf.string.name, value: "1", fields: new Map() }],
+									[
+										{
+											type: stringSchemaIdentifier,
+											value: "1",
+											fields: new Map(),
+										},
+									],
 								],
 								[
 									brand("baz"),
-									[{ type: leaf.number.name, value: 2, fields: new Map() }],
+									[{ type: numberSchemaIdentifier, value: 2, fields: new Map() }],
```

Comment:
its fine to leave these as is. The simple-tree schema for leaf nodes just wraps the leaf domain, so these values were fine.